src/Sarif.Viewer.VisualStudio/ErrorList/SarifSnapshot.cs

```diff
@@ -23,7 +23,6 @@ namespace Microsoft.Sarif.Viewer.ErrorList
     internal class SarifSnapshot : TableEntriesSnapshotBase, IWpfTableEntriesSnapshot
     {
         private readonly List<SarifErrorListItem> _errors;
```

Comment:
While we're in here: This class has a get-only property named `Errors` that is never assigned. It can't be needed, right?

Reply:
It's set in the ctor

Reply:
OH sorry, misread. Will remove